README.md

```diff
@@ -1,6 +1,12 @@
-This is a Formula Student Driverless Simulation Competition System.
+This is a Formula Student Driverless Simulation Competition System. (FSDS)
 It will provide a virtual environment where Autonomous Systems from different Formula Student teams can compete in time-trial challenges. 
-The competition will take place during the driverless event, FS-Online 2020.
+The first competition will take place during the driverless event, FS-Online 2020.
+
+FSDS is brought to you by your friends at Formula Student Team Delft, MIT Driverless and FSEast.
+
+Would you like to help develop this project? Create [an issues](https://github.com/FS-Online/Driverless-Competition-Simulator/issues) introducing yourself and we will get you up to speed.
```

Comment:
this seems a bit weird no? we have documentation about how to get set up and they are always free to open a PR and get it reviewed. If there is something unclear about the setup process then they can open an issue?

Reply:
you are right, removed.